apps/hannk/Makefile

```diff
@@ -6,26 +6,64 @@ include ../support/Makefile.inc
 # builds but easier to enable it for everything.
 CXXFLAGS += -Wno-unused-private-field -fno-exceptions -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wunused-variable -I$(MAKEFILE_DIR)
 
+BENCHMARK_OUT = benchmark
+ifeq (hexagon-32-qurt,$(findstring hexagon-32-qurt,$(HL_TARGET)))
+	# Building benchmark application as shared object instead of elf for
+	# hexagon target.
+	BENCHMARK_OUT := $(BENCHMARK_OUT).so
+	BENCHMARK_HEXAGON_FLAGS = -shared -fPIC -G0
+	HEXAGON_STUBS = $(BIN)/$(HL_TARGET)/stubs.o
+	TEST ?= test/misc/MUL.tflite
```

Comment:
The benchmark app already runs every benchmark passed to it on the command line. Maybe we could push the whole folder, and run test/*/* like the non-Hexagon tests?

I think all of this logic should be in the test-hexagon-sim/test-hexagon-device target BTW, not stored in environment variables like TEST (that is way too generic).

Reply:
Hey Dillon, using `test/*/*` is not working for me. Now I realized that run_main_on_hexagon has a max-limit of 2048 characters.

Reply:
We can run all tests one at a time maybe?

Reply:
I guess we have to. I think we should make all targets work this way though, not just Hexagon, so it is consistent.